purchase_triple_discount/models/purchase_order.py

```diff
@@ -19,15 +19,15 @@ def _add_supplier_to_product(self):
              for line in self.order_line.filtered('discount3')])
         return super(PurchaseOrder, self.with_context(
             discount2_map=discount2_map, discount3_map=discount3_map)
-            )._add_supplier_to_product()
+        )._add_supplier_to_product()
 
 
 class PurchaseOrderLine(models.Model):
     _inherit = "purchase.order.line"
 
     @api.depends('discount2', 'discount3')
     def _compute_amount(self):
```

Comment:
Add the comment for clarifying that this is only for changing depends of the computed